### ISSUE TITLE
Cannot type while JAWS is enabled on IE11

### ISSUE THREAD

**Issue by apksdk** [moroshko/react-autowhatever#52]
### Issue Description
JAWS 2019 (Screen Reader) cannot enter forms mode when you tab into the component on IE11. This means you cannot type at all in the component while the screen reader is open.

JAWS Version: 2019.1906.10
IE11 Version: 11.0.9600.19463

### Reproducible Demo
Editor URL: https://stackblitz.com/edit/react-9nd9ga
APP URL: https://react-9nd9ga.stackblitz.io/

### Steps to Reproduce
1. Launch JAWS 2019 and view the demo using the app url on IE11
2. Use the keyboard to navigate to the component

### Expected Behaviour
Expected a "bop" sound to be made, and can start typing in the input

### Actual Behaviour
The "bop" sound does not play, and you cannot type anything in the input

### Possible Solution
I've done some investigations and it appears to be because ``role=combobox`` is set in the container instead of the input. By moving the mentioned attribute and the ones related to it inside the input tag, it allows JAWS to correctly enter forms mode.

**Comment by Indimeco** [comment]
I have done a deep investigation and I believe this is most likely a JAWS defect https://github.com/FreedomScientific/VFO-standards-support/issues/221

react-autowhatever is following the spec set forth by WAI-ARIA 1.1 https://www.w3.org/TR/wai-aria-1.1/#combobox

**Comment by apksdk** [comment]
Closing this issue as it's not a problem caused by the component.
